src/main/java/org/opentripplanner/transit/model/framework/DeduplicatorNoop.java

```diff
@@ -0,0 +1,56 @@
+package org.opentripplanner.transit.model.framework;
+
+import java.util.BitSet;
+import java.util.List;
+import javax.annotation.Nullable;
+
+class DeduplicatorNoop implements DeduplicatorService {
```

Comment:
Is it wise to keep this class in `src/main` rather than in the test folder? The chance of someone using it by accident increases.

Reply:
I see that the issue store does the same but it's not used as a fallback if you pass null, as far as I'm aware.